app/models/story.rb

```diff
@@ -184,6 +184,16 @@ def touch_series
     series.try(:touch)
   end
 
+  def series=(series_object)
+    super
+    self.account_id = series.account_id if series&.account_id.present?
```

Comment:
yes! Per our conversation, I moved it into the model, since it's technically possible to imagine there are other contexts in which a series might be set, and that should _always_ set the account.